RELEASE-NOTES.md

```diff
@@ -183,7 +187,6 @@ Changes in the `Entity` hierarchy:
 * Changed the constructor signature of `Property`
 * Removed `Entity::setClaims` (`Item::setClaims` has been retained)
 * Removed `Entity::stub`
-* Removed `Property::newEmpty`
```

Comment:
`Property:newEmpty` was **not** removed in version 1.0, see https://github.com/wmde/WikibaseDataModel/blob/1.0.0/src/Entity/Property.php#L172.